src/buildings/nobMilitary.cpp

```diff
@@ -328,8 +329,14 @@ void nobMilitary::LookForEnemyBuildings(const nobBaseMilitary* const exception)
         {
             unsigned distance = gwg->CalcDistance(pos, (*it)->GetPos());
 
+            // check if military building is reachable
+            if(gwg->GetGGS().isEnabled(AddonId::FRONTIER_DISTANCE_REACHABLE)
+               && DoesReachablePathExist(*gwg, (*it)->GetPos(), pos, std::max(distance * 3, MILITARY_RADIUS[3])))
+            {
+                frontier_distance = 0;
```

Comment:
Err, wait. `frontier_distance = 0` means "far". So I think it should have been `!DoesReachablePathExist`, or did I miss anything?

Reply:
i definitely with agree you!  Every testcase should have a partner with addon disabled. i dont know why i added a explicit "addoninactive". i will rename them, to match a naming scheme.
I will refactor those first. also i recognized that there is a test case doubled. 
I will take a look into it in the evening.